### DISCUSSION TITLE
Ruby 3.2 bricks minimal mistakes?

### DISCUSSION THREAD

**Discussion by heesters** [mmistakes/minimal-mistakes#4090] [Q&A] [answered]
After the ruby 3.2 update I can't seem to get the theme working:
  Liquid Exception: undefined method `tainted?' for nil:NilClass in /vendor/ruby/3.2.0/gems/minimal-mistakes-jekyll-4.24.0/_layouts/single.html

Anyone else have this problem?

**Comment by heesters** [comment]
This is fixed by the latest Jekyll release.

**Comment by saiqulhaq** [comment, marked as answer]
thanks
fixed by upgrading jekyll to v4.3.2
